doc/indexing.rst

```diff
@@ -241,6 +241,15 @@ index labels along a dimension dropped:
 
 ``drop`` is both a ``Dataset`` and ``DataArray`` method.
 
+Use :py:meth:`~xarray.Dataset.drop_dims` to drop a full dimension.  Any variables
+with these dimensions are also dropped:
+
+.. ipython:: python
+
+    ds.drop_dims('time')
+
+``drop_dims`` is a ``Dataset`` method.
```

Comment:
Maybe delete this, and above say "to drop a full dimension from a dataset."?

Reply:
Done.